src/nxdoc/tutorials/transversal-how-tos/how-to-implement-local-groups-or-roles-using-computed-groups.md

```diff
@@ -146,6 +146,21 @@ In this how to, you will learn how to let managers of a workspace determine who
 
 {{/callout}}
 
+Starting from 8.10 HF-04 you need to add the following contribution.
+
+{{#> panel type='code'}}
+
+```xml
+<extension point="configuration" target="org.nuxeo.runtime.ConfigurationService">
```

Comment:
@michaelgena could you fixup this, please? Then this PR will be ready to be merged :+1: